src/Microsoft.PowerShell.Commands.Diagnostics/project.json

```diff
@@ -10,6 +10,30 @@
         "warningsAsErrors": true
     },
 
+    "configurations": {
+        "Linux": {
+            "buildOptions": {
+                "define": [ "UNIX" ],
+                "debugType": "portable"
+            }
+        },
+        "Debug": {
+                "buildOptions": {
+                    "debugType": "portable"
+                }
```

Comment:
You can see the `buildOptions` section is still not following the existing indentation patterns. The existing indentation pattern is to indent 4 spaces for any enclosing sections. Many other changes in project.json files also have the same issue. Could you please fix them all? Thanks!